### PR TITLE
Add README explaining the XML Schema.

### PR DESCRIPTION
**Pull request by Zambelli24**
Need 1+ people to review and tell me what is/is not clear. There are ??? in the file which are things that I could not find an explanation for. Those may need Kai and Bryan to provide clarification but if someone knows something feel free to mention that as well.

### CODE REVIEW

File: ClinicalTrialXMLREADME.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+### XML Conforms to this [Schema](https://clinicaltrials.gov/ct2/html/images/info/public.xsd)
+### Furhter explanation of important terms can be found [here](https://clinicaltrials.gov/ct2/about-studies/glossary)
+-
+* **clinical_study** - all data about the clinical trial stored under this field
+* **required_header** - header that provides general info about the download
+	* **download_date** - date the date was downloaded
+	* **link_text** - the text that would display instead of the url if opened in a webpage
+		* Says the same thing for each file
+	* **url** - where to find the data for the file online
+* **id_info** - different ways the classify the study id
```
Comment (LucianoJ1996) [inline ClinicalTrialXMLREADME.md]: should be "different ways to classify the study id"

Comment (LucianoJ1996) [inline ClinicalTrialXMLREADME.md]:
```suggestion
* **id_info** - different ways to classify the study id
```

File: ClinicalTrialXMLREADME.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Clinical Trial Data from ClinicalTrials.gov
+-
+## Quick explanatory guide of the possible fields in one XML file 
+### XML Conforms to this [Schema](https://clinicaltrials.gov/ct2/html/images/info/public.xsd)
+### Furhter explanation of important terms can be found [here](https://clinicaltrials.gov/ct2/about-studies/glossary)
+-
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: Miscellaneous bullet point

File: ClinicalTrialXMLREADME.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* **clinical_study** - all data about the clinical trial stored under this field
+* **required_header** - header that provides general info about the download
+	* **download_date** - date the date was downloaded
```
Comment (LucianoJ1996) [inline ClinicalTrialXMLREADME.md]:
```suggestion
	* **download_date** - date the data was downloaded
```

File: ClinicalTrialXMLREADME.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+* **source** - same as lead_sponsor agency
+* **oversight_info**
+	* **has_dmc** - whether or not the study has a data monitoring committee
+	* **is\_fda\_regulated_drug** - yes or no
+	* **is\_fda\_regulated_device** - yes or no
+	* **is\_unapproved\_device** - yes or no
+	* **is_ppsd** - ???
+	* **is\_us\_export** - yes or no
+* **brief_summary**
+	* **textblock** - short explaination of what the study is doing and looking to test
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: "explanation" instead of "explaination"

File: ClinicalTrialXMLREADME.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+* **detailed_description**
+	* **textblock** - more indepth explanation of the study, includes a lot more information than the breif summary
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: "brief" instead of "breif"

File: ClinicalTrialXMLREADME.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+* **overall\_status** - potential values depend on the study_type
+	* Study Type: Expanded Access - Values: Available, No Longer Available, Temporarily Not Available, Approved for Marketing
+	* Study Type: N/A - Values: Withheld
+	* Study Type: Anythig Else - Values: Active not recruiting, Completed, Enrolling by initiative, Not yet recruiting, Recruiting, Suspended, Terminated, Withdrawn
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: "anything" instead of "anythig"

File: ClinicalTrialXMLREADME.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+	* **allocation** - how the tests are administered to the patients
+		* Value: Randomized
+	* **intervention_model** - general design of the strategy for assigning treatments to participants
+	* **primary_purpose** - reason for the clinical trial
+		* Values: Treatment
+	* **masking** - tells who knows which people were treated
+* **primary_outcome**
+	* **measure** - important for evaluating the effect of the treatment, what it is supposed to do
+	* **time_frame** - ???
+* **number\_of\_arms** - number of group/sub-groups in the clinical trial
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: group(s)?

File: ClinicalTrialXMLREADME.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+* **enrollment** - number of participants in the clinical study tagged with a type
+	* Types: Actual, Anticipated, Estimate
+* **condition** - disease, disorder, etc being studied - can be multiple
+* **arm_group** 
+	* **arm\_group\_label**
+	* **arm\_group\_type**
+	* **description**
+* **intervention** - treatment
+	* **intervention_type** - type of treatment
+		* Values: Behavioral, Biological, Combination Product, Device, Diagnostic Test, Dietary Supplement, Drug, Genetic, Procedure, Radiation, Other
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: Should "Genetic, Procedure" have a comma in it?

File: ClinicalTrialXMLREADME.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+			* **country**
+* **location_countries** - countries the study is being conducted in
+	* **country**
+* **reference** - sources of information for study???
+	* **citation**
+* **verification_date** - most recent date that the study sponsor/investigator confirmed trial info is accurate
+* **study\_first\_submitted** - date of first submission to ClinicalTrials.gov
+* **study\_first\_submitted\_qc** - date when study was consistent with NLM quality control review criteria
+* **study\_first\_posted** - date the study first appeared on ClinicalTrials.gov
+* **results\_first\_submitted** - date the summary results were first submission to ClinicalTrials.gov
```
Comment (AndCarr) [inline ClinicalTrialXMLREADME.md]: "first submitted to ClinicalTrials.gov"?

File: ClinicalTrialXMLREADME.md (hunk 8 above)
Comment (Zambelli24) [reply ClinicalTrialXMLREADME.md]: Yes, they are two separate values.,

**Comment by Zambelli24** [comment]
2 pull requests accidentally got turned into 1. Here are links to the updated requests that separate adding the potential database schema and adding the readme explaining the XML schema of a clinical study.
[XML README](https://github.com/MoravianCollege/clashboard/pull/4)
[Database Schema](https://github.com/MoravianCollege/clashboard/pull/5)

**Pull request closed without merging**